### DISCUSSION TITLE
NFT details return value

### DISCUSSION THREAD

**Discussion by markopoloparadox** [capsule-corp-ternoa/chain#34] [General]
@ETeissonniere  @ybensacq 

Currently when the NFTs `details` method is called it always returns a valid value even when an invalid nft id is passed. Does it make sense to change its return type to `Option<>` or `Result<>` so that the response from an invalid  id can be distinguished from a valid id?

Btw, if this kind of questions are supposed to be communicated through the Issues mechanism or through private messaging please tell me so that I don't make a mistake next time.

**Comment by ETeissonniere** [comment]
I think issues would be best here. We can certainly modify this, though the default choice in this case would be to keep it as it is. Based on our use cases returning an `Option` would be optimal.
